models/model.js

```diff
@@ -9,4 +9,23 @@ exports.fetchTopics = () => {
         
         return result.rows;
     });
-};
\ No newline at end of file
+};
+
+exports.fetchArticleById = (article_id) => {
+    return db
+    .query(
+        `SELECT * FROM articles WHERE article_id = $1;`, [article_id]
+    )
+    .then((result) => {
+        const id = result.rows[0]
+        if(!id) {
+            return Promise.reject({
+                status: 404,
+                msg: "Unknown Id"
```

Comment:
I would say in this case, a message of something like "article not found" would be more appropriate but this is preference. Also i think naming your article data "id" is a bit misleading as its not the value of the article id. Also further down you return result.rows[0], when you have already assigned the value of results.rows[0] to id but then not referenced it in the return